templates/books-index.html

```diff
@@ -0,0 +1,10 @@
+<div ng-repeat="book in booksIndexCtrl.books" class="row">
+	<div class="col-xs-5 col-md-3"><img ng-src="{{ book.image }}" alt="" class="img-responsive"></div>
```

Comment:
If you aren't setting alt text, you don't need to include the attribute. It's best practice to include alt text, though!